test/extended/images/db_helpers.go

```diff
@@ -0,0 +1,120 @@
+package images
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	"k8s.io/kubernetes/pkg/util/wait"
+)
+
+// MySQL is a MySQL helper for executing commands
+type MySQL struct {
+	oc        *exutil.CLI
+	PodName   string
+	Container string
+	Env       map[string]string
+}
+
+// NewMysql queries OpenShift for a pod with given name, saving environment
+// variables like username and password for easier use.
+func NewMysql(oc *exutil.CLI, podName, masterPodName string) (*MySQL, error) {
+	m := &MySQL{
+		oc:      oc,
+		PodName: podName,
+	}
+
+	if pod, err := oc.KubeREST().Pods(oc.Namespace()).Get(podName); err != nil {
```

Comment:
you just need PodInterface to get the container name. No need to have full CLI here.